run_service.py

```diff
@@ -353,6 +353,13 @@ def get_local_config() -> MemeooorrConfig:
     )
 
     memeooorr_config.store()
+    
+    # set the environment variables according to the configuration
+    apply_env_vars({
+        'BASE_LEDGER_RPC': memeooorr_config.base_rpc,
+        'BASE_LEDGER_CHAIN_ID': memeooorr_config.home_chain_id,

```

Comment:
why is the var called "BASE"?

Reply:
because they should be set for this: https://github.com/dvilelaf/meme-ooorr/blob/main/packages/dvilela/services/memeooorr/service.yaml#L99:L100